sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -933,29 +932,37 @@ supported parameter combination is specified in the following table.
 
 [frame="none",options="header"]
 |======================
-| A and B type | C and D type | M | N | K | Minimum Compute Capability
-.3+| `matrix_type::fp16`  .3+| `matrix_type::fp32`
-|16 |16 |16 .6+| sm_70
+| A and B type | C type | D type | M | N | K | Minimum Compute Capability
+.3+| `matrix_type::fp16`  .3+| `matrix_type::fp32` .3+| `matrix_type::fp32`
+|16 |16 |16 .12+| sm_70
 |8 |32 |16
 |32 |8 |16
-.3+| `matrix_type::fp16`  .3+| `matrix_type::fp16`
+.3+| `matrix_type::fp16`  .3+| `matrix_type::fp16` .3+| `matrix_type::fp16`
 |16 |16 |16
 |8 |32 |16
 |32 |8 |16
-.3+| `matrix_type::sint8`  .3+| `matrix_type::sint32`
+.3+| `matrix_type::fp16`  .3+| `matrix_type::fp32` .3+| `matrix_type::fp16`
+|16 |16 |16
+|8 |32 |16
+|32 |8 |16
+.3+| `matrix_type::fp16`  .3+| `matrix_type::fp16` .3+| `matrix_type::fp32`
```

Comment:
BTW, where are the bfloat16 combinations?

Reply:
They are in the same table at the bottom, between tf32 and fp64.